modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc

```diff
@@ -23,13 +23,15 @@ endif::[]
 
 . On the right-hand side of the *Operator Details* page, select *Uninstall Operator* from the *Actions* drop-down menu.
 +
-An *Uninstall Operator?* dialog box is displayed, reminding you that:
+An *Uninstall Operator?* dialog box is displayed, alerting you that:
 +
 [.small]
 --
-*Removing the Operator will not remove any of its custom resource definitions or managed resources. If your Operator has deployed applications on the cluster or configured off-cluster resources, these will continue to run and need to be cleaned up manually.*
+*Operator VerticalPodAutoscaler will be removed from openshift-vertical-pod-autoscaler. Click on the checkbox below to also remove all Operands associated with this Operator. If your Operator configured off-cluster resources, these will continue to run and require manual cleanup.*
```

Comment:
Add the --> *The* Operator VerticalPodAutoscaler

Reply:
If possible, avoid using "will be"

Reply:
@skrthomas This is the pop up message that appears in the web console's dialog box, hence won't be able to make this correction. However, if needed we can follow up with the console team and track it separately.